apps/balancer-tools/src/app/apr/round/(components)/PoolListTable.tsx

```diff
@@ -145,20 +153,27 @@ function TableRow({
   network,
   tableData,
   setTableData,
+  throttleHandler,
 }: {
   poolId: string;
   roundId: string;
   network: number;
   tableData: PoolTableData[];
   setTableData: Dispatch<SetStateAction<PoolTableData[]>>;
+  throttleHandler: ThrottledFunction<Argument, ReturnValue>;
```

Comment:
Need help typing this

Reply:
why pass this as a prop?

Reply:
The `Table` instance includes `p-Throttle`, then we use `TableRow` to display data. This way enables `TableRow` to perform multiple queries without any constraints while the responsibility of limiting the queries is managed by `p-throttle` instanced on `Table`.